app/src/main/java/dmar/akan/akademiaandroida/features/data/remote/model/CharacterRemote.kt

```diff
@@ -28,4 +28,11 @@ data class CharacterRemote(
         episode = episode,
         url = url
     )
+
+    inner class CharacterOriginRemote {
```

Comment:
Kierunek jest bardzo dobry! Możesz zagnieździć data class w obrębie innej data class bez korzystania ze słowa kluczowego `inner`. W takim przypadku będzie to wyglądało w taki sposób: 
```
data class CharacterOriginRemote (
    val name: String,
    val url: String
)
```
Ważne jest, żeby dla każdej warstwy stworzyć taki model oddzielnie, czyli w domenie będzie obiekt `CharacterOrigin` wewnątrz klasy `Character`, a w warstwie prezentacji będzie `CharacterOriginDisplayable` wewnątrz klasy `CharacterDisplayable`.